### PR TITLE
Prevent some cases of pinging internal addresses

### PR DESCRIPTION
**Pull request by Konicai**
Bot can currently ping the test server right now.

### CODE REVIEW

**Comment by Konicai** [comment]
Depends on #253

**Review by Camotoy — APPROVED** [review]

File: src/main/java/org/geysermc/discordbot/commands/PingCommand.java [hunk 1]
```diff
@@ -109,6 +109,11 @@ private MessageEmbed handle(String ip) {
         String[] ipParts = ip.split(":");
 
         String hostname = ipParts[0];
+
+        if (DumpHandler.isInternalIP(hostname)) {
```
Comment (Camotoy) [inline src/main/java/org/geysermc/discordbot/commands/PingCommand.java:113]: Maybe move this to a generic location.

Comment (Konicai) [reply src/main/java/org/geysermc/discordbot/commands/PingCommand.java:113]: Agreed

Comment (Konicai) [reply src/main/java/org/geysermc/discordbot/commands/PingCommand.java]: Updated

File: src/main/java/org/geysermc/discordbot/util/NetworkUtils.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+ * @link https://github.com/GeyserMC/GeyserDiscordBot
+ */
+
+package org.geysermc.discordbot.util;
+
+import org.apache.commons.net.util.SubnetUtils;
+
+public class NetworkUtils {
+
+    public static final SubnetUtils.SubnetInfo[] INTERNAL_IP_RANGES = new SubnetUtils.SubnetInfo[] {
```
Comment (rtm516) [inline src/main/java/org/geysermc/discordbot/util/NetworkUtils.java:32]:
```suggestion
    private static final SubnetUtils.SubnetInfo[] INTERNAL_IP_RANGES = new SubnetUtils.SubnetInfo[] {
```

File: src/test/java/org/geysermc/discordbot/commands/search/InternalIpTest.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ * @author GeyserMC
+ * @link https://github.com/GeyserMC/GeyserDiscordBot
+ */
+
+package org.geysermc.discordbot.commands.search;
```
Comment (rtm516) [inline src/test/java/org/geysermc/discordbot/commands/search/InternalIpTest.java:26]: This should be src/test/java/org/geysermc/discordbot/util/NetworkUtilsTest.java

**Review by rtm516 — CHANGES_REQUESTED** [review]

**Review by Tim203 — APPROVED** [review]

**Pull request merged**